### PR TITLE
[Fix](Variant) check column index validation for extracted columns

### PR DESCRIPTION
**Pull request by eldenmoon**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

File: be/src/olap/rowset/segment_v2/inverted_index_writer.h [hunk 1]
```diff
@@ -70,6 +73,25 @@ class InvertedIndexColumnWriter {
 
     virtual void close_on_error() = 0;
 
+    // check if the column is valid for inverted index, some columns
+    // are generated from variant, but not all of them are supported
+    static bool check_column_valid(const TabletColumn& column) {
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_writer.h:78]: suggest name : check_type_support_index

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]

**Pull request merged**